README.rst

```diff
@@ -201,6 +201,14 @@ tools.
 
     $ python pipdeptree.py --json
 
+The dependency graph (acyclic if there are no circular dependencies)
```

Comment:
It seems to me that if the argument `tree` given to the function `dumps_graphviz_dot` is a `dict` that defines cyclic dependencies, then the resulting directed graph should contain a cycle. I can see that a warning about cyclic dependencies was added by https://github.com/Dunedan/pipdeptree/commit/ea17d2e5586e40e4f03079d9c9f450f1f25734c1#diff-c9a0cc375d0f079d6251af1de64f535bR394.